CRM/Upgrade/Incremental/php/FiveForty.php

```diff
@@ -97,4 +98,13 @@ public static function addContributionProductFK(CRM_Queue_TaskContext $ctx): boo
     return TRUE;
   }
 
+  /**
+   * Set the timezone to the default for existing Events.
+   */
+  public static function setEventTZDefault(CRM_Queue_TaskContext $ctx) {
+    // Set default for CiviCRM Events to user system timezone (most reasonable default);
+    $defaultTZ = CRM_Core_Config::singleton()->userSystem->getTimeZoneString();
+    CRM_Core_DAO::executeQuery('UPDATE `civicrm_event` SET `event_tz` = %1;', [1 => [$defaultTZ, 'String']]);
```

Comment:
It's not always possible but a goal for upgrade steps is to be re-runnable later. As-written this would wipe out any that had been set since the first time. Suggest adding `WHERE event_tz IS NULL`.